src/doc/nomicon/unbounded-lifetimes.md

```diff
@@ -11,7 +11,7 @@ lifetime can be regarded as `'static`.
 
 Almost no reference is `'static`, so this is probably wrong. `transmute` and
 `transmute_copy` are the two other primary offenders. One should endeavor to
-bound an unbounded lifetime as quick as possible, especially across function
+bind an unbounded lifetime as quickly as possible, especially across function
```

Comment:
I think `bound` is correct here. cf. "If an output lifetime is elided, then it must be bounded by an input lifetime." below.